### PR TITLE
btop: add Spanish translation

### PR DESCRIPTION
**Pull request by LaNataliaaa**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by gutjuri** [comment]
Thanks for your contribution! You have created a directory `btop.md/` and added a file `btop.md` inside it. Could you please edit your pull request such that the file you created is at the following location: `pages.es/common/btop.md`. Thank you!

**Comment by gutjuri** [comment]
I don't know either, maybe @kbdharun knows how to do it :P

File: pages.es/common/btop.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# btop
+
+> Un monitor de recursos que muestra información sobra la CPU, memoria, discos, red y procesos.
+> Versión en C++ de`bpytop`.
+> Más información: <https://github.com/aristocratos/btop>.
+
+- Iniciar`btop`:
```
Comment (acuteenvy) [inline pages.es/common/btop.md:7]: Missing spaces
```suggestion
> Versión en C++ de `bpytop`.
> Más información: <https://github.com/aristocratos/btop>.

- Iniciar `btop`:
```

**Review by acuteenvy — COMMENTED** [review]
> I don't know how to do that, I'm kinda noob with GitHub.

Move the file locally, `git add` it, commit and push. I already moved it for you.

**Comment by kbdharun** [comment]
> > I don't know how to do that, I'm kinda noob with GitHub.
> 
> Move the file locally, `git add` it, commit and push. I already moved it for you.

Yep this is the correct way.

An alternative to do it graphically here is to rename the file (to a different name btop1, to prevent name conflicts) and remove the redundant directory. Then in another commit you can fix it back to the correct name.

**Review by kant — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**